src/components/ButtonDiv.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable react/button-has-type */
```

Comment:
- [ ] Please do not disable linters, they help make your code clean and follow the standard. Therefore I suggest to try to fix the errors instead of disabling them.

**HIT:** You are getting `react-button-has-type` because your `button` tags do not have a `type` attribute. You can fix that by adding `type="button"` to each button as show below :point_down: 
![Screenshot from 2023-08-08 12-02-39](https://github.com/fatima-najafi/math-magicians/assets/91142432/2ee05b21-1a81-4801-b741-fc5def113d0d)